### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by ManuelFeller**
working on general documentation as well as a setup guide

### CODE REVIEW

File: documentation/setup.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Setup (and operation)
+
+This document contains all you need to know to run a simple donation campaign using this tool.
+
+***Don't be scared about it's length.** We tried to cover everything very detailed (so that you do not need to ask anyone if you have no deep knowledge in one part or the other) - and if you already have some knowledge you should be able to do this quite fast... ;)*
```
Comment (cschweikert) [inline documentation/setup.md:5]: 😂

File: documentation/README.md [hunk 2]
```diff
@@ -11,3 +11,6 @@ This section helps you to set everything up if you want to reuse this
 - Survey Monkey limits you to 500 API requests per day and 120 per minute; https://www.surveymonkey.de/r/2PK6JLP
   - you have no "API" to pull teh results outside of the Survey Monkey results page (at least not if you use the free version)
```
Comment (cschweikert) [inline documentation/README.md]:
```suggestion
  - you have no "API" to pull the results outside of the Survey Monkey results page (at least not if you use the free version)
```

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Manuel Feller
+Copyright (c) 2022 Manuel Feller & Christian Schweikert
```
Comment (cschweikert) [inline LICENSE:3]: ❤️

**Review by cschweikert — APPROVED** [review]
Very nice and very detailed!

File: documentation/setup.md (hunk 1 above)
Comment (ManuelFeller) [reply documentation/setup.md:5]: ;)

File: LICENSE (hunk 3 above)
Comment (ManuelFeller) [reply LICENSE:3]: Ehrensache

**Pull request merged**